src/core_plugins/kibana/public/context/query_parameters/actions.js

```diff
@@ -43,7 +46,13 @@ export function QueryParameterActionsProvider() {
     )
   );
 
+  const addTermsFilter = (state) => (field, values, operation) => {
+    const { queryParameters: { indexPatternId } } = state;
```

Comment:
This just stems from the attempt to be consistent with the structure of the other action creators that fetch more than one value from the state. You are correct in that it does not contribute to readability here.